libsolidity/ast/ASTAnnotations.h

```diff
@@ -236,6 +236,8 @@ struct UserDefinedTypeNameAnnotation: TypeNameAnnotation
 {
 	/// Referenced declaration, set during reference resolution stage.
 	Declaration const* referencedDeclaration = nullptr;
+	/// Whether a virtual lookup needs to be performed to find the declaration.
+	bool requiresVirtualLookup = false;
```

Comment:
Can this one ever be true?

Reply:
Good point. I worked backwards and looked where we do a `resolveVirtual` in the codegen and it seems that method didn't lead to a case where a `UserDefinedTypeName` was resolved..

Reply:
In fact, there is no mentioning of `UserDefinedTypeName` anywhere in `codegen/*`

Reply:
Yes, I think this is just a "type checking" concept. Codegen only uses `annotation().type`